integ-test/src/test/resources/indexDefinitions/datatypes_index_mapping.json

```diff
@@ -13,10 +13,14 @@
       "binary_value": {
         "type": "binary"
       },
-      "date_value":  {
+      "date_value": {
         "type" : "date",
         "format": "yyyy-MM-dd HH:mm:ss"
       },
+      "date_nanos_value": {
+        "type" : "date_nanos",
```

Comment:
Can you add tests without format being specified in the mapping? Test default formats as per specified in https://opensearch.org/docs/latest/field-types/supported-field-types/date-nanos/#:~:text=If%20you%20don%E2%80%99t%20specify%20the%20format%20when%20mapping%20a%20field%20as%20date_nanos%2C%20the%20default%20format%20is%20strict_date_optional_time%7C%7Cepoch_millis%20that%20lets%20you%20pass%20values%20in%20either%20strict_date_optional_time%20or%20epoch_millis%20format

Reply:
Fixed in 7a04f82b7, thanks. `date_nanos` with format tested in another test.